enforcer-parent/commons/src/main/java/org/wso2/choreo/connect/enforcer/commons/logging/CustomJsonLayout.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org).
```

Comment:
Change year as "2022"

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2633/commits/c3087100180167139f73a7fb351a91359c5d9bbf